stock_orderpoint_manual_procurement/models/stock_warehouse_orderpoint.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+from datetime import datetime
+from openerp.addons import decimal_precision as dp
+from openerp.tools import float_compare, float_round
+
+
+UNIT = dp.get_precision('Product Unit of Measure')
+
+
+class StockWarehouseOrderpoint(models.Model):
+    _inherit = 'stock.warehouse.orderpoint'
+
+    @api.multi
+    @api.depends("product_min_qty", "product_id", "qty_multiple")
+    def _compute_procure_recommended(self):
+        procurement_model = self.env['procurement.order']
+        for op in self:
+            op.procure_recommended_date = \
+                procurement_model._get_orderpoint_date_planned(
+                    op, datetime.today())
+            procure_recommended_qty = 0.0
+            prods = procurement_model._product_virtual_get(op)
+            if prods is None:
+                continue
+            if float_compare(prods, op.product_min_qty,
+                             precision_rounding=op.product_uom.rounding) < 0:
+                qty = max(op.product_min_qty, op.product_max_qty) - prods
```

Comment:
I've created a Reordering rule for the product [DVD] Blank DVD-RW with the following values:
Minimum Quantity 5.000 
Maximum Quantity 25.000
Quantity Multiple 1.000
Unit of measure: Dozens
The quantity on hand is 0.00 and the forecasted quantity is 0.00 dozens. 

When I create a procurement  and I select the product and then the unit of measure, the quantity proposed is the maximum quantity in the reordering rule and the unit of measure is Dozens.

It works as expected, however why do we choose the maximum instead of the minimum?